recipes/postgresql/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "9.5.3" %}
+
+package:
+    name: postgresql
+    version: {{ version }}
+
+source:
+    fn: postgresql-{{ version }}.tar.bz2
+    url: https://ftp.postgresql.org/pub/source/v{{ version }}/postgresql-{{ version }}.tar.bz2
+    sha256: 7385c01dc58acba8d7ac4e6ad42782bd7c0b59272862a3a3d5fe378d4503a0b4
+    patches:
+        - fix_libraries_win.patch  # [win]
```

Comment:
That is from @msarahan's PR. Not sure. (We are skipping Windows here though and I left this here just for "completeness" as we may need it in the feedstock.)